add loom video and cache fast follows! (#5486)
fishtown-analytics/docs.getdbt.com: website/docs/docs/use-dbt-semantic-layer/sl-cache.md

Declarative caching enables you to pre-warm the cache using [saved queries](/docs/build/saved-queries) by setting the cache config to `true` in your `saved_queries` settings. This is useful for optimizing performance for key dashboards or common ad-hoc query requests. For configuration details, refer to [Declarative caching setup](#declarative-caching-setup).

<details>

<summary>Check out this video demo to see how Declarative caching works</summary>

>   <Loom id="aea82a4dee364dfdb536e7b8068684e7" titl="This video demonstrates the concept of Declarative caching, how to run it using the dbt Cloud scheduler, and how fast your dashboards load as a result."/>

Author: ```suggestion
  <Loom id='aea82a4dee364dfdb536e7b8068684e7' titel="This video demonstrates the concept of Declarative caching, how to run it using the dbt Cloud scheduler, and how fast your dashboards load as a result." />
```

Declarative caching enables you to pre-warm the cache using [saved queries](/docs/build/saved-queries) by setting the cache config to `true` in your `saved_queries` settings. This is useful for optimizing performance for key dashboards or common ad-hoc query requests. For configuration details, refer to [Declarative caching setup](#declarative-caching-setup).

<details>

> <summary> 📹 Check out this video demo to see how Declarative caching works!</summary>

Reviewer: ```suggestion
<summary> 📹 Check out this video demo to see how declarative caching works!</summary>
```

Author: yea i was back and forth w it - the name of the feature is 'Declarative' so that's why i had it like this. but all good!

> This video demonstrates the concept of Declarative caching, how to run it using the dbt Cloud scheduler, and how fast your dashboards load as a result.

Reviewer: ```suggestion
This video demonstrates the concept of declarative caching, how to run it using the dbt Cloud scheduler, and how fast your dashboards load as a result.
```

Refer to the following diagram, which illustrates what happens when the dbt Semantic Layer receives a query request:

> <Lightbox src="/img/docs/dbt-cloud/semantic-layer/declarative-cache-query-flow.jpg" width="70%" title="Overview of the Declarative cache query flow" />

Author: i have an asana task for these diagrams already requested

> <Lightbox src="/img/docs/dbt-cloud/semantic-layer/declarative-cache-query-flow.jpg" width="70%" title="Overview of the Declarative cache query flow" />

Reviewer: ```suggestion
<Lightbox src="/img/docs/dbt-cloud/semantic-layer/declarative-cache-query-flow.jpg" width="70%" title="Overview of the declarative cache query flow" />
```

```
</File>

### Run your declarative cache

> After setting up declarative caching in your YAML configuration, you can now run [exports](/docs/use-dbt-semantic-layer/exports) with the dbt Cloud job scheduler to build a cached table from a saved query, into your data platform.

Reviewer: ```suggestion
After setting up declarative caching in your YAML configuration, you can now run [exports](/docs/use-dbt-semantic-layer/exports) with the dbt Cloud job scheduler to build a cached table from a saved query into your data platform.
```
